src/FileTemplates.jl

```diff
@@ -36,6 +36,28 @@ function new_table_migration(module_name::String, resource::String) :: String
   """
 end
 
+function new_table_migration(module_name::String, namesAndTypes::String ,resource::String)::String
```

Comment:
I don't think we need to put this migration generator just to create a migration for the test. Within the `test/` folder we can add a `migration/` folder where we add the already created migration. Then at the beginning of the test we perform the so called set-up phase, were we create the migrations table, and run this migration. At the end of the test we perform the so-called tear-down where we run the migrations down and delete the tests table.

Reply:
This is not for test cases. This function is meant to use for development of the whole configuration file from a given type. In a real world y don't want write the same thing twice, one time in a struct and one time for a table migration. With this function and some more it is possible to create the right definition once. If you don't want this in your package, I can remove this there and implement this for my own use somewhere else.

Reply:
Oh, I understand now. I think it's a great idea - so basically to generate the migration based on the model's fields. Yeah, that'd be awesome if we can make it work. In this case shouldn't we just pass an optional model type to the new_table_migration function and use the fields and types defined in the model?